src/webhook_launcher/app/views.py

```diff
@@ -145,172 +142,36 @@ def index(request):
-            if tag:
-                lsr.tag = tag
-
-        lsr.save()
-
-    # PUT / trigger webhook
-    def update(self, request, pk=None):
-        try:
-            hook = WebHookMapping.objects.get(pk=pk)
-        except WebHookMapping.DoesNotExist:
-            return Response(
-                {'cannot find webhook id': pk},
-                status=status.HTTP_404_NOT_FOUND
-            )
-
-        msg = hook.trigger_build()
-        return Response({'WebHookMapping Triggered by API': msg})
-
-    # PATCH / update webhook
-    def partial_update(self, request, pk=None):
-        try:
-            hook = WebHookMapping.objects.get(pk=pk)
-        except WebHookMapping.DoesNotExist:
-            return Response(status=status.HTTP_404_NOT_FOUND)
-        serializer = WebHookMappingSerializer(hook)
-
-        # first take the original data
-        patched_data = serializer.data
-
-        # patch keys from request.data
-        for key, value in request.data.items():
-            patched_data[key] = value
-
-        serializer = WebHookMappingSerializer(hook, data=patched_data)
-
-        lsr_data = request.data.get('lsr', None)
-        revision = request.data.get('revision', None)
-
-        if lsr_data or revision:
-            # in rare case there is no mapping to lsr from hook we create one
-            if not hook.lsr:
-                if revision:
-                    LastSeenRevision.objects.get_or_create(
-                        mapping=hook,
-                        revision=revision,
-                    )
-                else:
-                    return Response(
-                        {"detail": "no LastSeenRevision mapped to object. "
-                                   "Please give also 'revision'",
-                         },
-                        status=status.HTTP_400_BAD_REQUEST,
-                    )
-
-            lsr_serializer = LastSeenRevisionSerializer(
-                hook.lsr, data=lsr_data,
-            )
-            lsr_patch_data = lsr_serializer.data
-
-            if lsr_data:
-                for key, value in lsr_data.items():
-                    lsr_patch_data[key] = value
-            if revision:
-                lsr_patch_data['revision'] = revision
-
-            lsr_serializer = LastSeenRevisionSerializer(
-                hook.lsr,
-                data=lsr_patch_data,
-            )
-
-        if serializer.is_valid():
-            if lsr_data or revision:
-                if lsr_serializer.is_valid():
-                    serializer.save()
-                    lsr_serializer.save()
-            else:
-                serializer.save()
-        else:
-            return Response(
-                serializer.errors, status=status.HTTP_400_BAD_REQUEST)
-
-        return Response(serializer.data)
-
-    @detail_route(
-        methods=['get', 'put'],
-        permission_classes=[permissions.IsAuthenticatedOrReadOnly],
-    )
-    def find(self, request, obsname, project, package):
```

Comment:
Didn't notice to remove it from urls, will fix that. The view it self was removed because same thing can be done with the standard API. No point in maintaining two things to do the same thing